src/mzcloud-cli/src/main.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License in the LICENSE file at the
+// root of this repository, or online at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Command-line tool for interacting with Materialize Cloud.
+
+use mzcloud::apis::{
+    configuration::Configuration,
+    deployments_api::{
+        deployments_certs_retrieve, deployments_create, deployments_destroy, deployments_list,
+        deployments_logs_retrieve, deployments_retrieve, deployments_update,
+    },
+    mz_versions_api::mz_versions_list,
+    schema_api::schema_retrieve,
+};
+use mzcloud::models::{deployment_request::DeploymentRequest, deployment_size::DeploymentSize};
```

Comment:
Can you configure your rust analyzer import style to "last"? That's the style we've weakly settled on.

Reply:
Do you mean `importGranularity = "module"`?

Reply:
Oh, I guess so. Seems like they renamed that option.